plugins/org.eclipse.n4js.cli/pom.xml

```diff
@@ -0,0 +1,211 @@
+-->
+<project
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>org.eclipse.n4js.cli</artifactId>
+	<packaging>eclipse-plugin</packaging>
+
+	<parent>
+		<groupId>org.eclipse.n4js</groupId>
+		<artifactId>org.eclipse.n4js.plugins</artifactId>
+		<version>0.0.1-SNAPSHOT</version>
+		<relativePath>../pom.xml</relativePath>
+	</parent>
+
+	<properties>
+		<cli.jar-longname>${project.artifactId}-${project.version}-${cli.fat.jar.classifier}.jar</cli.jar-longname>
+	</properties>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-clean-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.xtend</groupId>
+				<artifactId>xtend-maven-plugin</artifactId>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<version>${maven-dependency-plugin.version}</version>
+				<executions>
+					<execution>
+						<id>copy-dependencies</id>
+						<phase>package</phase>
+						<goals>
+							<goal>copy-dependencies</goal>
+						</goals>
+						<configuration>
+							<outputDirectory>${project.build.directory}/lib</outputDirectory>
+							<overWriteReleases>false</overWriteReleases>
+							<overWriteSnapshots>false</overWriteSnapshots>
+							<overWriteIfNewer>true</overWriteIfNewer>
+							<excludeTransitive>true</excludeTransitive>
+							<excludeArtifactIds>
+								com.ibm.icu,
+								org.junit,
+								org.eclipse.jdt.core,
+								org.eclipse.jdt.debug,
+								org.eclipse.jdt.launching,
+								org.bouncycastle.bcpg,
+								org.bouncycastle.bcprov,
+								org.eclipse.jgit,
+								org.apache.sshd.osgi,
+								org.apache.sshd.sftp,
+								com.fasterxml.jackson.core.jackson-core,
+								com.fasterxml.jackson.core.jackson-databind,
+								com.fasterxml.jackson.core.jackson-annotations,
+						<!-- 		org.apache.ant,
+								org.apache.commons.lang,
+								org.apache.commons.logging,
+								org.eclipse.core.commands,
+								org.eclipse.core.contenttype,
+								org.eclipse.core.expressions,
+								org.eclipse.core.filesystem,
+								org.eclipse.core.jobs,
+								org.eclipse.core.resources,
+								org.eclipse.core.runtime,
+								org.eclipse.core.variables,
+								org.eclipse.debug.core,
+								org.eclipse.emf.codegen.ecore,
+								org.eclipse.emf.codegen,
+								org.eclipse.emf.mwe.core,
+								org.eclipse.emf.mwe.utils,
+								org.eclipse.emf.mwe2.lib,
+								org.eclipse.emf.mwe2.runtime,
+								org.eclipse.equinox.app,
+								org.eclipse.equinox.preferences,
+								org.eclipse.equinox.registry,
+								org.eclipse.text,
+								org.eclipse.xtend.typesystem.emf,
+								org.eclipse.xtend, -->
```

Comment:
Why are these not removed entirely (just commented out)?
Either remove entirely or add a brief comment why they might still be relevant in the future.